src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/CaseEventMessageMapperTest.java

```diff
@@ -17,8 +21,15 @@ class CaseEventMessageMapperTest {
     private static final LocalDateTime EVENT_TIME_STAMP = LocalDateTime.now();
     private static final LocalDateTime RECEIVED = LocalDateTime.now();
     private final ObjectMapper objectMapper = new ObjectMapper();
+    private CaseEventMessageMapper mapper;
 
-    private final CaseEventMessageMapper mapper = new CaseEventMessageMapper();
+    @BeforeEach
+    void setUp() {
+        mapper = new CaseEventMessageMapper(objectMapper);
+        objectMapper.registerModule(new Jdk8Module());
```

Comment:
Do we need to register these modules and configure? I have removed it and the test still passes.

Reply:
I was following uk.gov.hmcts.reform.wacaseeventhandler.config.JacksonConfiguration#objectMapper but you're right the test does still pass so I will remove it.